### PR TITLE
GRID-455-459-475-470: Resize last col & Drag Cell Selectionm & Dragging Last Column

### PR DESCRIPTION
**Pull request by Dwaynekj**
@joneit I intend to push an alpha tonight that won't be a 1.3.0 master target

### CODE REVIEW

File: src/lib/events.js [hunk 1]
```diff
@@ -341,9 +341,15 @@ module.exports = {
         var self = this;
 
         function handleMouseEvent(e, cb) {
-            var primitiveEvent = self.getGridCellFromMousePoint(e.detail.mouse),
+            var result = self.getGridCellFromMousePoint(e.detail.mouse),
```
Comment (joneit) [inline src/lib/events.js]: `result` traditionally used for return values....

**Comment by Dwaynekj** [comment]
@joneit  the only issue now is that the cursor state is wrong but the events work immediately
thats a separate ticket I saw u just assign to yourself

**Comment by Dwaynekj** [comment]
@joneit

File: src/renderer/index.js [hunk 2]
```diff
@@ -500,25 +500,46 @@ var Renderer = Base.extend('Renderer', {
...
+        var mousePoint = this.grid.newPoint(x - vc.left, y - vr.top),
+            cellEvent = new this.grid.behavior.CellEvent(vc.columnIndex, vr.index);
+
+        // cellEvent.visibleColumn = vc;
+        // cellEvent.visibleRow = vr;
+
+        result.cellEvent = Object.defineProperty(cellEvent, 'mousePoint', {value: mousePoint});
+
+        if (isPseudoCol || isPseudoRow) {
+            result.fake = true;
```
Comment (joneit) [inline src/renderer/index.js]: To resolve "lingering cursor" issue, add following after this line:

```javascript
this.grid.beCursor(null);
```

**Pull request merged**